src/views/modal_handler.rs

```diff
@@ -113,50 +143,90 @@ impl DisplayModal {
     }
 
     pub fn view(&self) -> Element<Message> {
-        let mut button_row = Row::new();
-        let mut text_style = TextStyle {
-            text_color: Color::new(0.447, 0.624, 0.812, 1.0),
-        };
-
-        if self.is_warning {
-            text_style = TextStyle {
-                text_color: Color::new(0.988, 0.686, 0.243, 1.0),
-            };
-            button_row = button_row.push(
-                button("Exit without saving")
-                    .on_press(Message::ModalHandle(ModalMessage::ExitApp))
-                    .style(MODAL_CANCEL_BUTTON_STYLE.get_button_style()),
-            ); // Exits the application
-            button_row = button_row
-                .push(
-                    button("Return to app")
+        match &self.modal_type {
+            Some(ModalType::Warning {
```

Comment:
I thought you might use the enum to specify what dialog to show, so

enum {
   HardwareDetails(could put dialog title here),
   etc
}

then the view could just be a match on that, and you might not need variables like "load_config" and others?